src/Infrastructure/RedisCustomerRepository.php

```diff
@@ -11,36 +11,20 @@ class RedisCustomerRepository implements CustomerRepositoryInterface
      * @Redis
      */
 
-    private $client;
-
-    public function connect()
-    {
-        try {
-            $this->client = new Predis\Client([
-                'scheme' => 'tcp',
-                'host' => '127.0.0.1',
-                'port' => 6379,
-                'database' => 0
-            ]);
-            echo 'Successfully connected to Redis';
-        } catch (Exception $e) {
-            echo 'Couldn\'t connect to Redis';
-            echo $e->getMessage();
-        }
-    }
-
     public function save(Customer $customer)
     {
-        $this->client->connect('127.0.0.1', 6379);
-        $this->client->set('customer', $customer);
-        $this->client->save();
-        $this->client->close();
+        $client = new \Redis();
```

Comment:
Intersted to see that you have moved from the PHP Redis library (Predis) to the PHP Redis extension. Not a bad move; out of interest why did you decide to do that?